styles.css

```diff
@@ -0,0 +1,77 @@
+body {
+  box-sizing: border-box;
+  font-family: 'poppins', sans-serif;
+  background: #e5e5e5;
+}
+
+.bio-container {
+  box-sizing: border-box;
+  width: 100%;
+  height: 730px;
+  border-radius: 0 0 0 100px;
+  background: #fff;
+}
+
+.toolbar {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 28px;
+  width: 100%;
+  background: #fff;
+}
+
+.hamburger-img {
+  padding-right: 36px;
+}
+
+.my-logo img {
+  padding-left: 24px;
+}
+
+.main {
+  box-sizing: border-box;
+  background: #fff;
+  background-image: url('./images/bg-mobile1.svg');
+  background-repeat: no-repeat;
+  background-size: contain;
+  padding-left: 24px;
+  padding-right: 24px;
+  margin-top: -50px;
+  width: 100%;
+  height: 625px;
+  border-radius: 0 0 0 100px;
+}
+
+.headline {
+  font-size: 40px;
+  font-weight: 700;
+  margin-bottom: 10px;
+  line-height: 52px;
+  padding-top: 120px;
+  color: #344563;
+}
```

Comment:
Please make sure your design is the same as the Figma template your header part color. according to the Figma template, the colour for this part is `#172B4D`. Kindly fix it.